services/core-api/tests/now_application_factories.py

```diff
@@ -476,4 +481,4 @@ class Params:
     #     if not isinstance(extracted, int):
```

Comment:
Remove commented-out code?

Reply:
not sure, there might be some excluded tests that are dependent on this, so I would rather keep it before we address that technical debt ticket first